src/flag_gems/runtime/backend/_nvidia/__init__.py

```diff
@@ -0,0 +1,22 @@
+import os
+import sys
+
+from .ops import *  # noqa: F403
+
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Dont's append other path to `sys.path` since it may cause name conflicts with other packages.

Reply:
Also, why do you need this?
It may be a sign that the dependencies between modules or packages are not properly organized.